lib/open_api_spex/schema.ex

```diff
@@ -869,6 +338,48 @@ defmodule OpenApiSpex.Schema do
 
   def properties(_), do: []
 
+  @doc """
+  Generate example value from a `%Schema{}` struct.
+
+  This is useful as a simple way to generate values for tests.
+
+  Example:
+
+      test "create user", %{conn: conn} do
+        user_request_schema = MyAppWeb.Schemas.UserRequestSchema.schema()
+        req_body = OpenApiSpex.Schema.example(user_request_schema)
+
+        resp_body =
+          conn
+          |> post("/users", req_body)
+          |> json_response(201)
+
+        assert ...
+      end
+  """
+  def example(%Schema{example: example} = schema) when not is_nil(example) do
+    schema.example
+  end
+
+  def example(%Schema{type: :object} = schema) do
+    Map.new(properties(schema), fn {prop_name, _} ->
+      property = schema.properties[prop_name]
+      example_value = example(property)
+      {prop_name, example_value}
+    end)
+  end
+
+  def example(%Schema{type: :array} = schema) do
+    item_example = example(schema.items)
+    [item_example]
+  end
+
+  def example(%Schema{type: :string}), do: ""
```

Comment:
Should we handle common [format](https://swagger.io/docs/specification/data-models/data-types/#string) values? Eg `date` and `date-time` would be handy.